bindings/protocols/coap/index.html

```diff
@@ -61,79 +61,130 @@ <h2>Protocol Overview</h2>
-            Parameters for blockwise transfer (as defined in [[RFC7959]]) can be set in a
-            <code>coap:blockwise</code> member.
-            The vocabulary terms are defined in full detail in section [[[#vocabulary]]].
+            The section below gives an overview of the features offered by CoAP at the time of writing
+            and provides examples of a how they can be used in a [[[WOT-THING-DESCRIPTION]]].
+            The vocabulary terms used in these examples are defined in section [[[#vocabulary]]] below.
         </p>
         <!-- Maybe RFC numbers could be included in the table? -->
-        <table class="def">
-            <thead>
-                <tr>
-                    <th>Feature</th>
-                    <th>Covered by</th>
-                </tr>
-            </thead>
-            <tbody>
-                <tr>
-                    <td>Request to Server</td>
-                    <td><code>"href": ..., "coap:method": ...</code></td>
-                </tr>
-                <tr>
-                    <td>Request to Forward Proxy</td>
-                    <td><code>"href": ..., "coap:method": ..., "coap:proxy": ...</code></td>
-                </tr>
-                <tr>
-                    <td>Caching: Validation</td>
-                    <td>n/a</td>
-                </tr>
-                <tr>
-                    <td>Conditional Requests</td>
-                    <td>n/a</td>
-                </tr>
-                <tr>
-                    <td>Content</td>
-                    <td><code>"coap:format": ...</code></td>
-                </tr>
-                <tr>
-                    <td>Content Negotiation</td>
-                    <td><code>"coap:accept": ...</code></td>
-                </tr>
-                <tr>
-                    <td>Observing Resources</td>
-                    <td><code>"subprotocol": "coap:observe"</code></td>
-                </tr>
-                <tr>
-                    <td>Block-Wise Transfer</td>
-                    <td><code>"coap:blockwise": { ... }</code></td>
-                </tr>
-                <tr>
-                    <td>Quick Block-Wise Transfer</td>
-                    <td><code>"coap:qblockwise": { ... }</code></td>
-                </tr>
-                <tr>
-                    <td>Hop Limit</td>
-                    <td><code>"coap:hoplimit": ...</code></td>
-                </tr>
-                <!--
-                <tr>
-                    <td>OSCORE</td>
-                    <td><code>SecurityScheme</code></td>
-                </tr>
-                <tr>
-                    <td>EDHOC</td>
-                    <td><code>SecurityScheme</code></td>
-                </tr>
-                -->
-            </tbody>
-        </table>
+        <section>
+            <h3>Requests</h3>
+            <p>
+                ... <code>href</code> and <code>coap:method</code>.
+            </p>
```

Comment:
Would something like this make sense?

```suggestion
            <p>
                CoAP protocol bindings are added to a form by using a CoAP
                URI scheme (c.f. section [[[#url]]]) in the <code>href</code>
                member. While each <code>op</code> type corresponds to a CoAP
                method code, implying a default value (c.f. section
                [[[#mapping]]]), a <code>coap:method</code> member can also be
                set explicitly to one of the CoAP methods, such as
                <code>GET</code>, <code>PUT</code>, or <code>POST</code>.
            </p>
```